### DISCUSSION TITLE
Mailing list for this package?

### DISCUSSION THREAD

**Discussion by Ompsda** [villekf/OMEGA#21] [Ideas]
Hi,

Is there a mailing list where the community can ask questions? Especially about the compilation and installation of the package.

Thank you,

**Comment by villekf** [comment]
At the moment no, since there hasn't been that much users either. For now, these GitHub discussions are probably closest to a mailing list.

However, now that you did mention it, I could investigate the possibility of setting up a mailing list sometime in the future.
